src/Compilers/CSharp/Test/Emit/CodeGen/StackAllocInitializerTests.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
```

Comment:
Started a list to capture follow-up work items: [https://github.com/dotnet/roslyn/issues/24699](https://github.com/dotnet/roslyn/issues/24699)
For instance, testing IOperations.

Reply:
I will add those tests in a follow-up PR.